src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java

```diff
@@ -146,11 +147,17 @@ public void process(BlobClient blobClient) {
         }
     }
 
-    private byte[] tryToDownloadBlob(BlobClient blobClient) throws IOException {
+    private byte[] downloadBlob(BlobClient blobClient) throws IOException {
         try (var outputStream = new ByteArrayOutputStream()) {
             blobClient.download(outputStream);
 
             return outputStream.toByteArray();
+        } catch (Exception exc) {
```

Comment:
This change doesn't make any difference in this class's logic - only intermediate exception is thrown, still to be caught in this class. No way to unit test this bit. Alternatively, I could extract this method into a separate dependency, but that's a bit much for such a trivial task.